lib/restaurant.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "texter"
+
+class Restaurant
+
+  def initialize(texter = Texter.new)
```

Comment:
Perhaps dependancy injection could have been used rather than initialising with a new Texter? Would be great to chat about why you decided to do it this way.